backend/server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/system/MultiQualityMode.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * SonarLint Core - Server API
+ * Copyright (C) 2016-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.serverapi.system;
+
+public enum MultiQualityMode {
```

Comment:
I think a better name here based on the enum values would be something like `SeverityMode` ? I looked in the SonarQube MMFs and there is no general name for this concept :/ 

Just saying because `MultiQualityMode` sounds like the values should be `true` or `false`, if that makes sense 🤷

Reply:
Ah you're probably right, multi quality mode is basically MQR mode if I understood correctly